src/Proto.Actor/LocalContext.cs

```diff
@@ -345,12 +344,8 @@ internal static Task DefaultSender(IContext context, PID target, MessageEnvelope
 
         private Task ProcessMessageAsync(object msg)
         {
-            Message = msg;
-            if (_receiveMiddleware != null)
-            {
-                return _receiveMiddleware(this);
-            }
-            return DefaultReceive(this);
+            _message = msg;
+            return _receiveMiddleware != null ? _receiveMiddleware(this) : DefaultReceive(this);

```

Comment:
`_receiveMiddleware?.Invoke(this) ??  DefaultReceive(this);` ?  

I guess I just hate tertiary operators so I'm being nitpicky.